cmd/objbench.go

```diff
@@ -188,6 +203,19 @@ func objbench(ctx *cli.Context) error {
 			uploadResult[1], uploadResult[2] = colorize("multi-upload", float64(total*5)/cost, cost, 2, tty)
 			uploadResult[1] += " MiB/s"
 			uploadResult[2] += " s/object"
+
+			logger.Infof("downloading the multipart upload file and checking its contents ...")
+			r, err := blob.Get(fname, 0, -1)
+			if err != nil {
+				logger.Fatalf("failed to get multipart upload file: %v", err)
+			}
+			cnt, err := io.ReadAll(r)
```

Comment:
This will be too slow for large test.

Maybe we should limit the size of multipart to 200MB?

Reply:
200MB is too small for the internal network, here I think you can add a download progress bar, which will give people a better experience.